M2/Macaulay2/m2/help.m2

```diff
@@ -291,9 +282,23 @@ documentationValue(Symbol, Package)         := (S, pkg) -> if pkg =!= Core then
 	    }))
 
 -----------------------------------------------------------------------------
--- Handling operators
+-- Details for developers
 -----------------------------------------------------------------------------
 
+linkToFile := (src, fn, pos) -> HREF { if installLayout === null then src else installLayout#"packages" | fn, fn | pos };
+
+-- TODO: support more objects
+getSource = method()
+getSource(Symbol, Thing)   := (S, x) -> null
+getSource(Symbol, Package) := (S, pkg) -> (
+    -- TODO: for packages, should we link to the location on GitHub instead?
```

Comment:
I think this is a probably a good idea.  Then we wouldn't need to mirror all the .m2 files on the website

Reply:
We also wouldn't need to deal with figuring out the path to source.